docs/configuration/misc.rst

```diff
@@ -44,3 +44,54 @@ Interval & duration fields in job definitions, pruning configurations, etc. must
 
     var durationStringRegex *regexp.Regexp = regexp.MustCompile(`^\s*(\d+)\s*(s|m|h|d|w)\s*$`)
     // s = second, m = minute, h = hour, d = day, w = week (7 days)
+
+
+.. _conf-include-directories:
+
+Including Configuration Directories
+-----------------------------------
+
+It is possible to distribute zrepl job definitions over multiple YAML files. This is
+achieved by using the `include` key in the configuration file.
+
+The directive is mutually exclusive with any other jobs definition and can only exist
+in the configuration file:
+
+::
+   global: ...
+
+   jobs:
+     include: jobs.d
+
+.. NOTE::
+   The included directory path is treated as absolute when starting with `/` else it
+   is treated as a relative path from the directory of the loaded configuration file.
```

Comment:
Regarding relative paths, I think it'd be more precise to state that the relative path is added to the `filepath.Dir()` of the main config file path used by the daemon. With a link to the list of default config locations [here](https://zrepl.github.io/configuration/overview.html#overview-terminology).